### DISCUSSION TITLE
Density effect corrections below 1 keV

### DISCUSSION THREAD

**Discussion by Jaap1957** [nrc-cnrc/EGSnrc#834] [Ideas]
Daar forum,

As Mr Tessier shows in #826, the "AE"  lower limit has a not negligible influence on energy spectrum results.
![afbeelding](https://user-images.githubusercontent.com/96918927/151336956-34918586-e6a0-4662-a456-bcfdfb5f36b4.png)

Part of this is due to the ESTAR table which lower end is 1 keV.
Here the first few points of this table, (i added (0,0) )
![afbeelding](https://user-images.githubusercontent.com/96918927/151337103-d7f43ba1-a872-4d62-81b7-e87b10b7d4ee.png)

It gives me the strong impression that a straight line from the lowest point to (0,0) would be a good extrapolation.

Would it be an idea to just extrapolate the ESTAR density correction factor from the lowest point to (0,0) ? That would remove 1 of the 4 low energy limits. I did not study this extensively but you are the experts, and maybe this is already a very old idea.

Nice code !

**Comment by ftessier** [comment]
I presume that EGSnrc indeed interpolates to zero (I would have to check in detail) to calculate the restricted stopping power between 0 and 1 keV (or more generally 0 to `ae`). But at any rate, as you can see in your graph at low energy the the density effect correction is pretty much insignificant; it is a relativistic effect.

The impact of `ae` on the spectrum is **not** related to the density effect correction (or lack of data point below 1 keV); it is simply due to the fact that secondary electrons below `ae` are never _generated_ in the in the first place!

**Comment by Jaap1957** [comment]
Maybe just discaard this idea then.

________________________________
